batch/sql/estimated-current.sql

```diff
@@ -244,12 +244,14 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   `cores_mcpu` INT NOT NULL,
   `status` TEXT,
   `n_pending_parents` INT NOT NULL,
+  `n_succeeded_parents` INT DEFAULT NULL,
```

Comment:
I was trying to make it so old records didn't have 0's for the n_succeeded_parents as that means something different than NULL to me and I thought it would be more error prone.

Reply:
ah that's a good point, but does feel a bit hacky to leave like this forever. I feel like this deserves its own migration of INT DEFAULT NULL -> set to proper value -> INT NOT NULL. Can we do this?

Reply:
We theoretically can, but I was worried about how to make this efficient and plan the migration so it's fast, idempotent, and correct. I guess my original feeling was the work wasn't worth it for a field that's only used when a job hasn't completed yet, but if you feel differently, I can try and work out what exactly needs to happen to do the migration.

Reply:
@daniel-goldstein Can I get input on this? Do you want to add a migration?

Reply:
Ya I think ultimately this will be a useful field, not just for this particular use case, so would prefer to introduce it the right way.